src/simtools/applications/derive_photon_electron_spectrum.py

```diff
@@ -74,6 +74,20 @@ def _parse(label):
         default=42.0,
         required=False,
     )
+    config.parser.add_argument(
+        "--scale_afterpulse_spectrum",
+        help="Scale afterpulse spectrum by the given factor",
+        type=float,
+        default=1.0,
+        required=False,
+    )
+    config.parser.add_argument(
+        "--threshold_afterpulse_spectrum",
+        help="Threshold for afterpulse ratio calculation",
```

Comment:
Ratio calculation?

Reply:
(note this parameter is replace in #1419, so no need to discuss here. This is the threshold used for the ratio of afterpulse to prompt calculation.